website/src/main/tut/docs/rules/DisableUnless.md

```diff
@@ -1,62 +0,0 @@
----
-layout: docs
```

Comment:
Instead of removing this file we should add a note saying this rule has been merged with `Disable` and include migration instructions. People may have linked to this page so we should not return 404.